cmd/ci-images-mirror/main.go

```diff
@@ -267,3 +278,9 @@ func mirrors(action string, n int, ms quayiociimagesdistributor.MirrorStore) (an
 		return nil, fmt.Errorf("invalid action: %s", action)
 	}
 }
+
+func execute(c *quayiociimagesdistributor.MirrorConsumerController) {
+	if err := c.Run(); err != nil {
+		logrus.WithError(err).Error("Error running")
```

Comment:
> Is it enough to just log the error and move forward?

For now as a starter it should be enough. I do not know how the errors hit us yet.
I am thinking retries, and add `--force` etc there.
If none of them work, logging error for human debugging.
Plus, the tool is watching the imagestreams, when it starts, it will pipeline all the tags overall again. So that we do not miss.

> Will the distributor still be useful without a consumer?
The consumer is https://github.com/openshift/ci-tools/pull/3627/files#diff-d20fe4d5bdebdd0e898bb1b5a9cb21ce5d075f0285eb619022c277a9b68ca1b1R87